### ISSUE TITLE
Flatpak does not have access to the icc color profiles folder.

### ISSUE THREAD

**Issue by alexb3d** [flatpak/flatpak#1660]

**Comment by alexb3d** [comment]
Ok, I had not seen this answer before going to twitter :¬)

In some applications (Inkscape, Darktable Scribus), Can not change the directory. It has predefined.
You can not package all icc profiles because some are proprietary. 
Some profiles are customized, mainly when calibrating (graphic) the hardware.

If the new directory is /run/host/usr/share/color/icc. You must warn the applications that use icc profiles to change it by default. Or that allow to select another directory as is the case of Gimp, Krita Rawtherapee.

**Comment by Jehan** [comment]
Hi @alexlarsson!
A small comment here. Shouldn't setting "--filesystem=/usr/share/color/icc" permission in the flatpak be enough to mount the host's /usr/share/color/icc into the same path in the sandbox?

**Comment by alexlarsson** [comment]
@Jehan I don't think that will work because there is no /usr/share/color/icc in the runtime that we can bind-mount the host directory on, and /usr is read-only so we can't create one.

**Comment by Jehan** [comment]
@alexlarsson I see. Well my alternative for now is that I added a build-time option `--with-icc-directory` in GIMP itself, allowing to override this hardcoded default. Then I'll build the flatpak with this option.
See https://github.com/flathub/org.gimp.GIMP/issues/15

This works because we were already creating a bookmark (which was broken in the flatpak) to this quite common directory in all file dialogs to search for color profiles. It is not perfectly ideal though since it won't be enough for cases where the person knows this folder by heart and just try to go there by writing its path while not paying attention to an existing bookmark.

I'm not sure how "standard" it is (i.e. I don't think it is explicitly written in the FHS, and maybe not in any Freedesktop spec or other), but this folder is kind of a de-facto standard. Simply all graphics software which have color-profile features need to have access to it and all distributions (that I know of) which install color profiles as packages will do so in this folder (on Fedora for instance, check out `dnf whatprovides /usr/share/color/icc/*.icc`).
If you make web search on the web, you'll find various references to this path as the path where every Free Software will bring people to look for standard profiles. For instance: https://wiki.scribus.net/canvas/Getting_and_installing_ICC_profiles

Do you think it would be possible to have a special permission for sandbox which wants read access to it? This is quite important for pro-graphics usage.
